tests/Schema/SchemaTest.php

```diff
@@ -479,6 +445,23 @@ public function testReferencingByUnqualifiedNameAmongQualifiedNamesWithDefaultNa
         self::assertFalse($schema->hasTable('public.s'));
     }
 
+    public function testGetNamespaces(): void
+    {
+        $schema = new Schema();
+
+        $schema->createTable('public.t');
+        self::assertEquals(['public'], $schema->getNamespaces());
+
+        $schema->createSequence('public.s');
+        self::assertEquals(['public'], $schema->getNamespaces());
+
+        $schema->dropTable('public.t');
+        self::assertEquals(['public'], $schema->getNamespaces());
+
+        $schema->dropSequence('public.s');
+        self::assertEmpty($schema->getNamespaces());
```

Comment:
Should we be more explicit about why we expect what we expect?

```suggestion
        self::assertEmpty(
                $schema->getNamespaces(),
                'Dropping all objects inside a schema should result in the schema being dropped as well.',
        );
```